html/changelogs/1138hahalaserdude.yml

```diff
@@ -0,0 +1,40 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#################################
+
+# Your name.  
+author: Scheveningen
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+ - balance: "Tentatively buffs laser damage types across the board. Makes them more useful."
```

Comment:
Imho that line (37) is enough to detail what has changed.
Let them figure out the rest ingame.

Reply:
i've changed the changelog three times already. do i have to do it again?

Reply:
Yes, our changelogs are usually short and to the point and not used to document every mechanical difference, but rather point people towards the changes so they know something has changed. 
(In this case the changelog would even be longer than the actual changes)
The interested parties can always look up the github pull requests to see what exactly has changed.

Reply:
only because i want to get this pushed into dev as fast as possible mmkay